runtime/features/data.rs

```diff
@@ -168,6 +168,14 @@ pub static FEATURE_DESCRIPTIONS: &[UnstableFeatureDescription] = &[
     config_option: ConfigFileOption::SameAsFlagName,
     env_var: Some("DENO_UNSTABLE_SLOPPY_IMPORTS"),
   },
+  UnstableFeatureDescription {
+    name: "subdomain-wildcards",
+    help_text: "Enable unstable subdomain wildcards for `--allow-net`",
+    show_in_help: false,
```

Comment:
I think we misunderstood each other - for now we should show it in help, I meant to flip that flag in a follow up PR after this PR lands, so we could enable wildcards by default and hide the flag in v2.4